Base/usr/share/man/man4/shadow.md

```diff
@@ -0,0 +1,26 @@
+## Name
+
+passwd - Encrypted passwords
```

Comment:
I personally would have liked something more descriptive here, like "System's users and password information"

Reply:
right, and i also just realized that's the wrong name lol